src/rustllvm/PassWrapper.cpp

```diff
@@ -435,14 +439,23 @@ extern "C" void LLVMRustConfigurePassManagerBuilder(
   unwrap(PMBR)->SLPVectorize = SLPVectorize;
   unwrap(PMBR)->OptLevel = fromRust(OptLevel);
   unwrap(PMBR)->LoopVectorize = LoopVectorize;
+
+#ifndef PGO_AVAILABLE
+  assert(!PGOGenPath && !PGOUsePath && "Should've caught earlier");
+#endif
+
   if (PGOGenPath) {
     assert(!PGOUsePath);
+#ifdef PGO_AVAILABLE
     unwrap(PMBR)->EnablePGOInstrGen = true;
     unwrap(PMBR)->PGOInstrGen = PGOGenPath;
+#endif
   }
   if (PGOUsePath) {
     assert(!PGOGenPath);
+#ifdef PGO_AVAILABLE
     unwrap(PMBR)->PGOInstrUse = PGOUsePath;
+#endif
```

Comment:
I think this would be easier to read if it were wrapped in a single `#ifdef`:
```cpp
#ifndef PGO_AVAILABLE
   assert(!PGOGenPath && !PGOUsePath && "Should've caught earlier");
#else
   if (PGOGenPath) {
     assert(!PGOUsePath);
     unwrap(PMBR)->EnablePGOInstrGen = true;
     unwrap(PMBR)->PGOInstrGen = PGOGenPath;
   }
   if (PGOUsePath) {
     assert(!PGOGenPath);
     unwrap(PMBR)->PGOInstrUse = PGOUsePath;
   }
#endif
```